.git-blame-ignore-revs

```diff
@@ -1,2 +1,4 @@
 # format all files
-f0acc811374c1249f9c4940ffb6b84f1b10d5011
\ No newline at end of file
+f0acc811374c1249f9c4940ffb6b84f1b10d5011
```

Comment:
totally outta curiosity, what does this do?

Reply:
it should mean that the mass format is ignored by the git blame, so it's easier to see what happened because formats don't matter in that context